packages/manager/modules/pci/src/projects/project/data-processing/notebooks/add-notebook/command/command.controller.js

```diff
@@ -0,0 +1,5 @@
+export default class CommandCtrl {
+  cancel() {
+    return this.goBack();
+  }
+}
```

Comment:
**nitpick:** Controller seems not required. You can directly use `goBack` in HTML.

Reply:
Fixed